lib/inc/cpp-pcp-client/connector/connector.hpp

```diff
@@ -159,8 +159,8 @@ class LIBCPP_PCP_CLIENT_EXPORT Connector {
     ///
     /// Throws a connection_not_init_error in case the connection has
     /// not been opened previously.
-    /// Also rethrows any exception that can be thrown by
-    /// startMonitoring.
+    /// Rethrows a possible exception previously caught by the
```

Comment:
In light of this, will `startMonitoring` actually throw most of the exceptions we mention? Or will they only be thrown from `stopMonitoring`?